### PR TITLE
Fix finished session judge

### LINKED ISSUE #305: [Assigned] All sessions are ended
## Overview (Required)
-
screenshot_20180119-163505

### PR DESCRIPTION
**Pull request by nappannda**
## Issue
- close #305

## Overview (Required)
- Fix session finished judge

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by konifar** [comment]
👀

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/item/SpeechSessionItem.kt [hunk 1]
```diff
@@ -41,6 +41,6 @@ data class SpeechSessionItem(
         val cal: Calendar = Calendar.getInstance(TimeZone.getTimeZone("JST"))
         cal.setTimeInMillis(System.currentTimeMillis())
         val now: Date = cal.getTime()
-        return now.time.toInt() < session.endTime.getTime().toInt()
+        return now.time.toInt() > session.endTime.getTime().toInt()
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/item/SpeechSessionItem.kt]: 😱

**Comment by konifar** [comment]
Many thanks! LGTM 👍

**Comment by konifar** [comment]
It would be better to add the test code 📝

**Comment by takahirom** [comment]
👀

**Comment by takahirom** [comment]
When i implement AlarmManager. Probably if you ues toInt , it become overflow.😭
If you have time can you check it?🙏

**Comment by panpanini** [comment]
`getTime()` should return a unix timestamp (millis) right? 
Android integer is 32bit, so max value is `2147483647`
that translates to 01/19/2038 - so I think overflow shouldn't happen. (Is my reasoning correct? 🤔 )

**Comment by takahirom** [comment]
mmm I will check it

**Pull request merged**